### PR TITLE
Adding talks, tutorials and more data

### PR DESCRIPTION
**Pull request by yabellini**

### CODE REVIEW

File: content/events/2024-11-20-LatinR.md [hunk 1]
```diff
@@ -35,10 +43,48 @@ resources: # can be added later, one entry per talk (don't add while still empty
 
 Don’t miss the [seventh edition of LatinR!](https://latinr.org/en/) This year, this year we will gather virtually to learn about the latest developments in R in Latin America. The program includes practical workshops, talks, and dialogue sessions with presenters.
 
-[Will Landau](/author/will-landau/) is one of the keynote speakers at LatinR!
+### Keynote
+
+[Will Landau](/author/will-landau/) is one of the **keynote speakers** at LatinR! with the talk "targets: a rediscovery from a different field"
```
Comment (maelle) [inline content/events/2024-11-20-LatinR.md:48]:
```suggestion
[Will Landau](/author/will-landau/) is a **keynote speaker** at LatinR! with the talk "targets: a rediscovery from a different field"
```

File: content/events/2024-11-20-LatinR.md [hunk 2]
```diff
@@ -35,10 +43,48 @@ resources: # can be added later, one entry per talk (don't add while still empty
...
+
+The practice of Statistics creates non-statistical challenges whose solutions come from software engineering. Version control, portable package libraries, and literate programming are already well-known examples. Pipeline tools, however, are an unlooked-for answer to workflow management struggles people seldom realize they have. The {targets} package is a pipeline tool that scales to large workloads, saves time, and embraces the comfort of your local R environment. This talk showcases {targets} tackling a formidable real-world example from Bayesian data analysis, and it posits a trove of unconventional breakthroughs waiting to be noticed by statisticians.
```
Comment (maelle) [inline content/events/2024-11-20-LatinR.md:50]: link to targets' pkgdown website?

File: content/events/2024-11-20-LatinR.md [hunk 3]
```diff
@@ -35,10 +43,48 @@ resources: # can be added later, one entry per talk (don't add while still empty
...
+
+### Satellite events
+
+* [Tradução + Hackathon = Traslaton rOpenSci](https://www.eventbrite.cl/e/traducao-hackathon-traslaton-ropensci-tickets-1045507530167) with [Pedro Faria](/author/pedro-faria/), [Francesca Palmeira](/author/francesca-belem-lopes-palmeira/), [Beatriz Milz](/author/beatriz-milz/), and [Yani Bellini Saibene](/author/yanina-bellini-saibene/): we will get together to learn about rOpenSci and how to contribute to the [Portuguese translation project](https://github.com/orgs/ropensci/projects/7). Monday, november 18th. 10:00 to 12:00 UTC-3.
```
Comment (maelle) [inline content/events/2024-11-20-LatinR.md:54]: add a timeanddate dot com link to find the time in one's timezone?

File: content/events/2024-11-20-LatinR.md [hunk 4]
```diff
@@ -35,10 +43,48 @@ resources: # can be added later, one entry per talk (don't add while still empty
...
+
+* Usando Quarto Live para tutoriales interactivos. [Yani Bellini Saibene](/author/yanina-bellini-saibene/).
+
+* Creando un universo mejor. [Mauro Lepore](/author/mauro-lepore/).
+
+* GuGuDaDados: A space for children and babies in events. [Haydee Svab](/author/haydee-svab/), [Beatriz Milz](/author/beatriz-milz/), Tatyane Paz, Ana Carolina Moreno
+
+* argendataR: la cocina de los datos de Argendata. [Juan Pablo Ruiz Nicolini](/author/juan-pablo-ruiz-nicolini/), Juan Gabriel Juara
+
+* inventarioRRNN. Un paquete de datos en R para abrir datos de recursos naturales de La Pampa. Juan Caldera, [Yani Bellini Saibene](/author/yanina-bellini-saibene/).
```
Comment (maelle) [inline content/events/2024-11-20-LatinR.md:80]: and this PR is called inventarioCHARLAS :joy_cat:

**Review by maelle — COMMENTED** [review]
wow so many things :star_struck:

File: content/events/2024-11-20-LatinR.md (hunk 2 above)
Comment (yabellini) [inline content/events/2024-11-20-LatinR.md:50]:
```suggestion
The practice of Statistics creates non-statistical challenges whose solutions come from software engineering. Version control, portable package libraries, and literate programming are already well-known examples. Pipeline tools, however, are an unlooked-for answer to workflow management struggles people seldom realize they have. [The {targets} package](https://docs.ropensci.org/targets/) is a pipeline tool that scales to large workloads, saves time, and embraces the comfort of your local R environment. This talk showcases {targets} tackling a formidable real-world example from Bayesian data analysis, and it posits a trove of unconventional breakthroughs waiting to be noticed by statisticians.
```

File: content/events/2024-11-20-LatinR.md (hunk 3 above)
Comment (yabellini) [inline content/events/2024-11-20-LatinR.md:54]:
```suggestion
* [Tradução + Hackathon = Traslaton rOpenSci](https://www.eventbrite.cl/e/traducao-hackathon-traslaton-ropensci-tickets-1045507530167) with [Pedro Faria](/author/pedro-faria/), [Francesca Palmeira](/author/francesca-belem-lopes-palmeira/), [Beatriz Milz](/author/beatriz-milz/), and [Yani Bellini Saibene](/author/yanina-bellini-saibene/): we will get together to learn about rOpenSci and how to contribute to the [Portuguese translation project](https://github.com/orgs/ropensci/projects/7). Monday, November 18th. 10:00 to 12:00 UTC-3.  ([find your time](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Navigating+the+R+ecosystem+using+R-universe+%28rOpenSci+Comm+Call%29&iso=20241118T10&p1=51&ah=1))
```

File: content/events/2024-11-20-LatinR.md [hunk 5]
```diff
@@ -35,10 +43,48 @@ resources: # can be added later, one entry per talk (don't add while still empty
...
+
+### Satellite events
+
+* [Tradução + Hackathon = Traslaton rOpenSci](https://www.eventbrite.cl/e/traducao-hackathon-traslaton-ropensci-tickets-1045507530167) with [Pedro Faria](/author/pedro-faria/), [Francesca Palmeira](/author/francesca-belem-lopes-palmeira/), [Beatriz Milz](/author/beatriz-milz/), and [Yani Bellini Saibene](/author/yanina-bellini-saibene/): we will get together to learn about rOpenSci and how to contribute to the [Portuguese translation project](https://github.com/orgs/ropensci/projects/7). Monday, november 18th. 10:00 to 12:00 UTC-3.  
+
+* **RDevDay@LatinR** with [Pao Corrales](/author/pao-corrales/), [Heather Turner](/author/heather-turner/), [Beatriz Milz](/author/beatriz-milz/), [Andrea Gomez Vargas](/author/andrea-gomez-vargas/), and [Luis Verde Arregoitia](/author/luis-verde-arregoitia/). Two session on Monday 18th and Tuesday 19th at 17:30 to 20:30 UTC -3.
```
Comment (yabellini) [inline content/events/2024-11-20-LatinR.md:56]:
```suggestion
* **RDevDay@LatinR** with [Pao Corrales](/author/pao-corrales/), [Heather Turner](/author/heather-turner/), [Beatriz Milz](/author/beatriz-milz/), [Andrea Gomez Vargas](/author/andrea-gomez-vargas/), and [Luis Verde Arregoitia](/author/luis-verde-arregoitia/). Two sessions on Monday 18th and Tuesday 19th at 17:30 to 20:30 UTC -3.  ([find your time](https://www.timeanddate.com/worldclock/fixedtime.html?msg=RDevDay%40LatinR&iso=20241118T1730&p1=51&ah=1))
```

**Pull request merged**